smartdashboard/views.py

```diff
@@ -149,6 +149,7 @@ def __init__(
         self.status_element = DeltaGenerator()
         self.experiment = experiment
```

Comment:
do you need to set experiment here as `self.experiment` if you call `super().__init__(view_model=exp)`? Can you avoid an extra reference and just use `self.view_model`?

Reply:
fixed!